src/pages/buy-sell/screens/BuySellHeader/__tests__/BuySellHeader.spec.tsx

```diff
@@ -47,6 +28,19 @@ jest.mock('@/hooks', () => ({
     },
 }));
 
+const mockStore = {
+    filteredCurrency: 'IDR',
+    selectedPaymentMethods: [],
+    setFilteredCurrency: jest.fn(),
+    setSortByValue: jest.fn(),
+    shouldUseClientLimits: false,
+    sortByValue: 'rate',
+};
+
+jest.mock('@/store', () => ({
+    useStore: jest.fn(() => mockStore),
+}));
```

Comment:
you don't always need to mock the store, only if you need to change the value or check if a setter function has been called